crates/swc_ecma_transforms_react/src/jsx/mod.rs

```diff
@@ -468,6 +472,8 @@ where
                 };
 
                 let mut key = None;
+                let mut __source = None;
```

Comment:
I think variable name should not use `_` as a prefix as it has special meaning in rust.

Reply:
👍  7fcafd2

I replaced `__source` with `source_props`.